lib/backend/pgbk/pgbk.go

```diff
@@ -173,14 +151,10 @@ func NewWithConfig(ctx context.Context, cfg Config) (*Backend, error) {
 	}
 
 	log := logrus.WithField(teleport.ComponentKey, componentName)
+	logger := slog.Default().With(teleport.ComponentKey, componentName)
```

Comment:
Do we really want to _add_ a `slog`-based logger instead of converting the `logrus` one?

Reply:
#41636 stacks on top of this PR and converts `pgbk` and `pgevents` to `slog` logging.